src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+@use '../utils/variables' as *;
+@use '../utils/mixins' as *;
+
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  width: 200px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  padding: 16px;
+
+  font-family: Roboto, sans-serif;
+
+  &__picture {
+    width: 160px;
+    height: 134px;
+    margin-top: $margin-top;
+  }
+
+  &__item-model {
+    font-weight: $medium-font-weight;
+    font-size: 12px;
+    line-height: $bigger-line-height;
+    color: rgba(6, 11, 53, 1);
+
+    box-sizing: border-box;
+    width: $paragraph-width;
```

Comment:
The variable `$paragraph-width` is used here. Verify that this variable is defined in the `variables` file or elsewhere in the project.